deployments/mainnet/migrations/1659582050_raise_supply_caps_and_seed_reserves_and_rewards.ts

```diff
@@ -0,0 +1,124 @@
+import { CometInterface, CometProxyAdmin, CometRewards, Configurator, ERC20, GovernorSimple, IGovernorBravo, ProxyAdmin, SimpleTimelock } from '../../../build/types';
```

Comment:
this file will need to be in `deployments/mainnet/usdc/` now that #498 is merged

Reply:
Rebased